### PR TITLE
std.algo_comp: updates alias syntax inside

### PR DESCRIPTION
**Pull request by burner**

### CODE REVIEW

File: std/algorithm/comparison.d [hunk 1]
```diff
@@ -1225,17 +1225,17 @@ MaxType!T max(T...)(T args)
 {
     //Get "a"
     static if (T.length <= 2)
-        alias args[0] a;
+        alias a = args[0];
     else
         auto a = max(args[0 .. ($+1)/2]);
-    alias typeof(a) T0;
+    alias T0 = typeof(a) ;
```
Comment (yebblies) [inline std/algorithm/comparison.d]: Extra space before `;`

Comment (burner) [inline std/algorithm/comparison.d]: fixed

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**